modules/op-pipelines-concepts.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id='op-pipelines-concepts_{context}']
+= {pipelines-title} concepts
+
+{pipelines-title} provide the following set of standard Custom Resource Definitions (CRDs) that act as the building blocks from which you can assemble a CI/CD pipeline for your application:
```

Comment:
I would suggest s/:/. for this statement since it's not followed by a list, but a series of subheaders.